src/main/java/org/opentripplanner/middleware/bugsnag/jobs/BugsnagEventHandlingJob.java

```diff
@@ -94,8 +102,9 @@ private void refreshEventRequest(BugsnagEventRequest request) {
                 Persistence.bugsnagEventRequests.removeById(request.id);
             }
             default: {
-                // Request not completed by Bugsnag yet. Do nothing and await the next cycle/refresh.
-                // TODO: Update the request data in the database? What if the status has changed since the last fetch?
+                // Request not completed by Bugsnag yet. Update the event request to record new status (this may not have
+                // changed) and await the next cycle/refresh.
+                Persistence.bugsnagEventRequests.replace(request.id, refreshedRequest);
```

Comment:
Nice catch!

Reply:
Nice catch! Updated to include the `projectId` and `daysInPast`.